TestCases/py_wrapper/flatPlate_rigidMotion/launch_flatPlate_rigidMotion.py

```diff
@@ -104,9 +104,8 @@ def main():
   # Extract the initial position of each node on the moving marker
   CoordX = np.zeros(nVertex_MovingMarker)
   CoordY = np.zeros(nVertex_MovingMarker)
-  CoordZ = np.zeros(nVertex_MovingMarker)
   for iVertex in range(nVertex_MovingMarker):
-    CoordX[iVertex], CoordY[iVertex], CoordZ[iVertex] = SU2Driver.GetInitialMeshCoord(MovingMarkerID, iVertex)
+    CoordX[iVertex], CoordY[iVertex] = SU2Driver.GetMarkerInitialCoordinates(MovingMarkerID, iVertex)
```

Comment:
why did we lose the 3D?

Reply:
The older `GetInitialMeshCoord()` required all 3 components, even for a 2D problem. Our `GetMarkerInitialCoordinates()` accounts for the problem definition and only returns 2 components in this case